azure/sdk-identity-resources-storage/Program.cs

```diff
@@ -0,0 +1,137 @@
+
+            Console.WriteLine("Press any key to continue and delete the resources...");
+            Console.ReadKey(true);
+
+            await DeleteResourceGroup(resourcesManagementClient, resourceGroupName);
+
+            return;
+        }
+
+        private static async Task DeleteResourceGroup(ResourcesManagementClient resourcesManagementClient, string resourceGroupName)
+        {
+            Console.WriteLine($"Deleting resource group {resourceGroupName}...");
+            ResourceGroupsDeleteOperation deleteOperation = await resourcesManagementClient.ResourceGroups.StartDeleteAsync(resourceGroupName);
+            await deleteOperation.WaitForCompletionAsync();
+            Console.WriteLine("Done!");
+        }
+
+        private static async Task<string> CreateStorageAccount(StorageManagementClient storageManagementClient, string rgName)
+        {
+            Console.WriteLine("Creating a new storage account...");
+
+            string storageAccountName = await GetStorageAccountName(storageManagementClient);
+            StorageAccountCreateParameters parms = new StorageAccountCreateParameters(
+                        new Sku("Standard_LRS"),
+                        Kind.BlobStorage,
+                        "West US");
+            parms.AccessTier = AccessTier.Hot;
+
+            StorageAccountsCreateOperation createStorageAccount =
+                await storageManagementClient.StorageAccounts.StartCreateAsync(rgName, storageAccountName, parms);
+                        await createStorageAccount.WaitForCompletionAsync();
+            Console.WriteLine($"Done creating account {storageAccountName}.");
+
+            return storageAccountName;
+        }
+
+        private static async Task<string> GetStorageAccountName(StorageManagementClient storageManagementClient)
+        {
+            bool foundAvailableName = false;
+            string storageAccountName = string.Empty;
+
+            while (!foundAvailableName)
+            {
+                storageAccountName = RandomName("storage", 20);
+                Response<CheckNameAvailabilityResult> availability =
+                    await storageManagementClient.StorageAccounts.CheckNameAvailabilityAsync(
+                        new StorageAccountCheckNameAvailabilityParameters(storageAccountName));
+
+                if (availability.Value.NameAvailable.GetValueOrDefault())
+                {
+                    foundAvailableName = true;
+                }
+            }
+            return storageAccountName;
+        }
+
+        private static async Task UploadBlobToStorageAccountUsingClientConnectionString(StorageManagementClient storageManagementClient, string resourceGroupName, string storageName)
+        {
+            Console.WriteLine("Creating a container and uploading a blob...");
+
+            string uploadFileName = "dotnet-bot.png";
+            string containerName = "images";
```

Comment:
Nit - make these [local `const string`s](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/const#example-1) to show best practices in samples.